pallets/connectors/src/lib.rs

```diff
@@ -100,19 +72,27 @@ pub type MessageOf<T> =
 
 pub type CurrencyIdOf<T> = <T as Config>::CurrencyId;
 
+pub type GeneralCurrencyIndexType = u128;
+
+pub type GeneralCurrencyIndexOf<T> =
+	GeneralCurrencyIndex<GeneralCurrencyIndexType, <T as pallet::Config>::GeneralCurrencyPrefix>;
+
 #[frame_support::pallet]
 pub mod pallet {
 	use cfg_primitives::Moment;
-	use cfg_traits::{Permissions, PoolInspect, TrancheCurrency};
+	use cfg_traits::{
+		CurrencyInspect, Investment, InvestmentAccountant, InvestmentCollector,
+		InvestmentProperties, Permissions, PoolInspect, TrancheCurrency,
+	};
 	use cfg_types::{
 		permissions::{PermissionScope, PoolRole, Role},
-		tokens::{CustomMetadata, GeneralCurrencyIndex},
+		tokens::{ConnectorsWrappedCurrency, CustomMetadata},
 	};
-	use frame_support::{error::BadOrigin, pallet_prelude::*, traits::UnixTime};
+	use frame_support::{pallet_prelude::*, traits::UnixTime};
 	use frame_system::pallet_prelude::*;
 	use pallet_xcm_transactor::{Currency, CurrencyPayment, TransactWeights};
 	use sp_runtime::traits::{AccountIdConversion, Zero};
-	use xcm::latest::OriginKind;
+	use xcm::{latest::OriginKind, v3::MultiLocation};
```

Comment:
I think you can just import `MultiLocation` from `latest`?
```suggestion
	use xcm::{latest::{OriginKind, MultiLocation};
```